apps/web/lib/plain/plainChat.tsx

```diff
@@ -69,26 +70,37 @@ interface PlainChatConfig {
 }
 
 const PlainChat = () => {
+  if (typeof window === "undefined") return null;
+
```

Comment:
You shouldn't use this

Reply:
React docs mention that 🔴 Do not call Hooks after a conditional return statement  here https://react.dev/warnings/invalid-hook-call-warning

Reply:
oh yes hooks should be first, this should be below L79